lighthouse-core/lib/manifest-parser.js

```diff
@@ -505,6 +505,7 @@ function parse(string, manifestUrl, documentUrl) {
     raw: string,
     value: manifest,
     warning: manifestUrlWarning,
+    manifestUrl: manifestUrl,
```

Comment:
I think `url` would be a better name here, considering the natural way to name a variable holding this object is `manifest` (gather-runner actually uses `parsedManifest` which is also cool), and `manifest.manifestUrl` is redundant.

I think `manifest.url` would not lead to any confusion about what the url is referring to.

Reply:
Do you think we should do this for the audit items value too? details.items[0].url vs details.items[0].manifestUrl. manifestUrl does add some clarity, although not sure if its necessary given that the audit refers to the manifest.

Reply:
`url` would be great there too

Reply:
There is some pretty good self documenting value for keeping it `manifestUrl` when it's deep in `installableManifest.details.items[0]`, though, since we don't publish docs for specific audit's details.

Note also it's a [term from the spec](https://www.w3.org/TR/appmanifest/#processing), to differentiate from "document URL" and `start_url`.